docs/MPURegionsUsageGuide.md

```diff
@@ -152,6 +193,10 @@
 		```c
 		up_mpu_set_register(regs);
 		```
+	6. If number of board specific regions to be reserved for vendor is to be added, modify/add the following API call after board MPU initialization:
+	```
+	mpu_set_nregion_board_specific(num);
+	```
```

Comment:
Is this point necessary here?
This example is supposed to be for adding new RO region only. 
And we have already mentioned that mpu_set_nregion_board_specific() must be called during boot up.